controller/reviewController.js

```diff
@@ -0,0 +1,78 @@
+const {
+  createReview,
+  updateReview,
+  findReviewById,
+} = require("../service/review");
+const {
+  updateLatestToiletPaperInfoById,
+  addReviewtoToilet,
+} = require("../service/toilets");
+
+exports.getReview = async (req, res, next) => {
+  try {
+    const reviewId = req.params.id;
```

Comment:
추후 백엔드 관련 리팩토링할때, 

router에서 ~/:reviewId 로 변경하고, 

```suggestion
    const {reviewId} = req.params;
```

이런식으로 구조분해 할당해서 가져오면 깔끔해지고 semantic하게 구현할것 같아요.
(파라미터 가져오는 부분 전부 협의후 맞춰야 할것 같네요!)

Reply:
네 추후 리팩토링 할때 통일 시키도록 하죠.

Reply:
아 괄호 사이에 띄어쓰기가 추가되야겠네요.
```suggestion
    const { reviewId } = req.params;
```